extensions/windows-event-log/tests/ConsumeWindowsEventLogTests.cpp

```diff
@@ -238,6 +238,18 @@ TEST_CASE("ConsumeWindowsEventLog extracts some attributes by default", "[onTrig
   auto logger_processor = test_plan->addProcessor("LogAttribute", "logger", Success, true);
   test_plan->setProperty(logger_processor, LogAttribute::FlowFilesToLog.getName(), "0");
 
+  SECTION("XML output") {
+    CHECK(test_plan->setProperty(cwel_processor, ConsumeWindowsEventLog ::OutputFormat.getName(), "XML"));
```

Comment:
I think these three should be `REQUIRE`s instead of `CHECK`s, because it doesn't make sense to continue running the test if we could not set the property

Reply:
You are right. I've switched to `REQUIRE` in https://github.com/apache/nifi-minifi-cpp/pull/1482/commits/efb169ad2b661a82f1608e80031dfcb7672eab06#diff-a2d991cd055149f79da7e62be4848113893ed6fc7104d9972c97cb671d4de7ceR242